.github/workflows/library-release.yml

```diff
@@ -0,0 +1,77 @@
+name: Build and Release
+
+on:
+  push:
+    tags:
+      - 'v*' # Trigger on version tags (e.g., v1.0.0)
+  workflow_dispatch: # Allow manual triggering of the workflow
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: 
+          - ubuntu-latest
+          - ubuntu-latest
+          #- macos-latest
```

Comment:
Why have the duplicated `macos-latest` instance commented out?

Reply:
For the mac I build arm, x86 and universal libraries. In order to build the universal one I need both builds on the same machine. So I only need one mac where I can cross compile to the other build and then combine them into a universal. I distribute all three libraries.